js/init_game.js

```diff
@@ -31,5 +38,6 @@ export const initGame = () => {
     { coordinate: head, direction: 'right' },
     { coordinate: body, direction: 'right' },
     { coordinate: tail, direction: 'right' },
+    { coordinate: food, direction: 'right' }
```

Comment:
@yang-xue I think I may do something like
```
{
  cat: [{ coordinate: ..., direction: ...}, ...]
  food: { coordinate: ..., direction: ... }
}
```
which will probably make it easier to read